experimental/packages/opentelemetry-instrumentation/package.json

```diff
@@ -72,9 +72,9 @@
   },
   "dependencies": {
     "@types/shimmer": "^1.0.2",
-    "import-in-the-middle": "1.3.5",
-    "require-in-the-middle": "^7.1.0",
-    "semver": "^7.3.2",
+    "import-in-the-middle": "^1.4.0",
```

Comment:
We should do this in a separate PR; this adds support for Node 20 (i think) and needs testing/docs updates, and is somewhat unrelated to the PR title as it's the only use of this package -> therefore no need to align with anything. :slightly_smiling_face:

Reply:
Makes sense. Reverted.